cli/src/bin.ts

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env node
+
+import { Command } from "commander";
+import * as path from "path";
+import fg from "fast-glob";
+import { watch } from "chokidar";
+import * as fs from "fs";
+import { promisify } from "util";
+import { parseMachinesFromFile } from "xstate-parser-demo";
+import {
+  doesTsTypesRequireUpdate,
+  getRangeFromSourceLocation,
+  makeXStateUpdateEvent,
+  writeToTypegenFile,
+} from "xstate-vscode-shared";
+
+const readFile = promisify(fs.readFile);
+const writeFile = promisify(fs.writeFile);
```

Comment:
```suggestion
const readFile = fs.promises.readFile;
const writeFile = fs.promises.writeFile;
```

Reply:
We are already targeting node 14 here so we can just do this:
```js
import fs from 'fs/promises'
```

Reply:
Didn't suggest that, as I thought he might be using something in `fs` that isn't available in `fs/promises`